### PR TITLE
[15.0][MIG] sale_operating_unit: Migration to 15.0

### PR DESCRIPTION
**Pull request by BT-apadron**

### CODE REVIEW

**Comment by JuanyDForgeflow** [comment]
@BT-apadron please check test

**Comment by MiquelRForgeFlow** [comment]
@BT-apadron Please add https://github.com/OCA/operating-unit/pull/540.

File: sale_operating_unit/test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo-addon-account_operating_unit @ git+https://github.com/OCA/operating-unit/pull/420/head#subdirectory=setup/account_operating_unit
```
Comment (alan196) [inline sale_operating_unit/test-requirements.txt:1]: Could you remove this file, these modules are already merged

**Review by alan196 — CHANGES_REQUESTED** [review]

**Comment by alan196** [comment]
@BT-apadron Do you plan to continue with this PR?

If you are OK I can continue with this PR

**Comment by BT-apadron** [comment]
@alan196 I can't right now but I can find some time this week. Anyway, if you want you can go ahead.

**Comment by AaronHForgeFlow** [comment]
Superseded here: https://github.com/OCA/operating-unit/pull/580

**Pull request closed without merging**